packages/transform-to-vanilla/src/types/style-rule.ts

```diff
@@ -2,3 +2,6 @@ import type { ComplexStyleRule, StyleRule } from "@vanilla-extract/css";
 
 export type ComplexCSSRule = ComplexStyleRule;
 export type CSSRule = StyleRule;
+
+export type VanillaStyleArray = Exclude<ComplexStyleRule, StyleRule>;
+export type VanillaClassNames = Exclude<VanillaStyleArray[number], StyleRule>;
```

Comment:
Can i know the purpose about this process? Especially line 3-4.

Reply:
i see, then can i understand the reason why we dont use `ComplexStyleRule` & `StyleRule` directly is for maintaining the original?